src/selectField/SelectField.style.tsx

```diff
@@ -36,8 +36,9 @@ export const StyledSelectField = styled.div`
     outline: 0;
     box-shadow: none;
     border: none;
+    border-radius: ${radius.l};
     cursor: pointer;
-    z-index: 1;
+    z-index: 0;
```

Comment:
Need the same deep position on the select as the arrow icon. Otherwise, in case of error display the select red background hide the icon.